tests/lib/rules/no-export-in-script-setup.js

```diff
@@ -143,6 +160,13 @@ ruleTester.run('no-export-in-script-setup', rule, {
       export const Foo = {}
       export enum Bar {}
       export {}
+      </script>
+      `,
+      output: `
+      <script setup lang="ts">
+      
+      
+      
       </script>
```

Comment:
Again, only the `export` keywords should be removed. Maybe the last statement (`export {}`) can be removed completely, but what about something more complex (e.g. `export { foo: 'bar' }`)? So I'd prefer a suggestion over an autofix at least in this case, too.